stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    from sklearn.model_selection import cross_val_score
</pre><p>Jennifer Ding:</p><ul><li>Would be nice to explain why you decided to use sklearn and the specific models as your ML models of choice</li></ul><p>Joe Cerniglia:</p><p>I might want to mention, as an explanation, that I am still a beginner to machine learning models, having worked with them for only the last 9 months. Therefore, I used the models I knew.&nbsp;I considered saying this in my early drafts, but it seemed a little too self-deprecating.&nbsp;I did use scientific method in selecting the best models among those I tried, but I did not select every model available.&nbsp;Take neural networks, for instance.&nbsp;I did not select them because I have not yet learned them. I will learn about them early next year when I begin taking the Machine Learning Certificate at Cornell University. From what I have studied, the models I selected as best are very popular for multi-class classification problems, even in preference sometimes to neural networks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>